nims-notes-things/src/App.js

```diff
@@ -1,25 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import Form from "./Form";
+import Header from "./Header";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header type="h1">Nims Notes Things</Header>
```

Comment:
what is type doing here?

Reply:
it doesn't do anything - when I coded it I thought type = h1 was a prop but i've realised that it is not a property because in my style component <Header>  is style.div 

I will be taking it out